### ISSUE TITLE
False-Positive: Lodash/Underscore Templates

### ISSUE THREAD

**Issue by sergei-startsev** [dequelabs/axe-linter-action#50]
Axe-linter does not recognize lodash/underscore HTML template files, resulting in false positive issues.

### Example 1
The lodash template below causes a false-positive `aria-valid-attr-value - Ensures all ARIA attributes have valid values` error:

```html
<li role="tab" aria-selected="<%-param1 %>" aria-controls="<%- param2 %>">Tab Name</li>
```

### Example 2
The lodash template below causes a false-positive `list - Ensures that lists are structured correctly` error:

```html
<ul>
    <% for (let i = 0; i < array.length; i++){%>
         <!--some code goes here-->
    <% } %>
</ul>
```

**Comment by ballendq** [comment]
Please see [comment in different ticket](https://github.com/dequelabs/axe-linter-action/issues/49#issuecomment-2422512725).

To replay the comment:

> Hi @sergei-startsev the linter does not support template files, this is why you are seeing problems.
>
> To reduce noise in results, we recommend you exclude files using the [exclude option](https://docs.deque.com/linter/4.0.0/en/axe-linter-configuration#exclude) within your configuration file.
